modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -832,7 +832,8 @@ private ObjectDefinition _addObjectDefinition(
 		storageType = Validator.isNotNull(storageType) ? storageType :
 			ObjectDefinitionConstants.STORAGE_TYPE_DEFAULT;
 
-		_validateEnableComments(enableComments, storageType, system);
+		_validateEnableComments(
```

Comment:
@mateussgn

can we pass here the unmodifiableSystemObject property instead of the modifiable and system property?